packages/webpack-cli/bin/cli.js

```diff
@@ -4,13 +4,21 @@
 require('v8-compile-cache');
 const importLocal = require('import-local');
 const runCLI = require('../lib/bootstrap');
+const { yellow } = require('colorette');
+const { packageExists, promptInstallation } = require('@webpack-cli/package-utils');
```

Comment:
Do we add `package-utils` in deps?

Reply:
Yes Already present.

https://github.com/webpack/webpack-cli/blob/5bae7977949f0e78df60e795eae171a17f4677e4/packages/webpack-cli/package.json#L26

Reply:
But these utils are only being used by `webpack-cli`, So we can safely move it from `package-utils` to `utils`. Let's do this in separate PR.

Reply:
https://github.com/webpack/webpack-cli/pull/1822 will solve this